### PR TITLE
Added notes for how to use Terminal and chocolatey to setup on Windows

### PR DESCRIPTION
**Pull request by JCash**

### CODE REVIEW

File: engine/dlib/src/test/wscript [hunk 1]
```diff
@@ -122,7 +126,7 @@ def build(bld):
     create_test(bld, 'test_spinlock', extra_libs = ['THREAD'])
     create_test(bld, 'test_sys', extra_libs = ['THREAD'], extra_defines = extra_defines)
     create_test(bld, 'test_template', extra_libs = ['THREAD'])
-    create_test(bld, 'test_ssdp_internals', extra_libs = ['THREAD'])
+    create_test(bld, 'test_ssdp_internals', extra_libs = ['THREAD'], skip_run = skip_ssdp)
```
Comment (JCash) [inline engine/dlib/src/test/wscript:129]: Not sure why it doesn't succeed on my machine :/

File: README_SETUP_WINDOWS.md [hunk 2]
```diff
@@ -4,9 +4,32 @@
...
+
+Once this is done, you can install new packages which are added to the PATH, by running:
+
+```sh
+choco install <package_name>
+```
+
 ### Required Software - Java JDK 11
 
-You need Java JDK 11 installed to build the tools. [Download and install release 11.0.1 from OpenJDK](https://jdk.java.net/archive/). When Java is installed you may also add need to add java to your PATH and export JAVA_HOME:
```
Comment (britzl) [inline README_SETUP_WINDOWS.md:9]: "add need add"

File: README_SETUP_WINDOWS.md [hunk 3]
```diff
@@ -4,9 +4,32 @@
...
+You need Java JDK 11 installed to build the tools. [Download and install release 11.0.1 from OpenJDK](https://jdk.java.net/archive/). 
+
+Ìnstall using Chocolatey:
+
+```sh
+choco install openjdk11
+```
+
+When Java is installed you may also add need to add java to your PATH and export JAVA_HOME:
```
Comment (britzl) [inline README_SETUP_WINDOWS.md:32]: typo: "add need add"

File: README_SETUP_WINDOWS.md [hunk 4]
```diff
@@ -53,21 +94,32 @@ You also need to install `wget`. From the mingw terminal run:
 
 #### Git
 
-You need to [download](https://git-scm.com/download/win) a command line version of Git. During install, select the option to not do any CR/LF conversion. If you use ssh (public/private keys) to access GitHub then:
+You need to [download](https://git-scm.com/download/win) a command line version of Git.
+
+During install, select the option to not do any CR/LF conversion.
+
+Before
```
Comment (britzl) [inline README_SETUP_WINDOWS.md:101]: Before?

**Review by britzl — DISMISSED** [review]

**Pull request merged**